jani_generator/src/jani_generator/jani_entries/jani_variable.py

```diff
@@ -18,35 +18,35 @@
 """
 
 from typing import Optional, Union, get_args
-from jani_generator.jani_entries import JaniExpression, JaniValue
 
 from as2fm_common.common import ValidTypes
+from jani_generator.jani_entries import JaniExpression, JaniValue
 
 
 class JaniVariable:
     def __init__(self, v_name: str, v_type: ValidTypes,
-                 v_init_value: Optional[Union[JaniExpression, JaniValue]] = None,
+                 init_value: Optional[Union[JaniExpression, JaniValue]] = None,
                  v_transient: bool = False):
-        assert v_init_value is None or isinstance(v_init_value, (JaniExpression, JaniValue)), \
+        assert init_value is None or isinstance(init_value, (JaniExpression, JaniValue)), \
             "Init value should be a JaniExpression or a JaniValue"
-        if v_init_value is not None:
-            if isinstance(v_init_value, JaniExpression):
-                self._init_value = v_init_value
+        self._init_expr: Optional[JaniExpression] = None
+        if init_value is not None:
+            if isinstance(init_value, JaniExpression):
+                self._init_expr = init_value
```

Comment:
JaniExpression can be constructed from a JaniExpression, so we can simply do:
```suggestion
        if init_value is not None:
              self._init_expr = JaniExpression(init_value)
```

Reply:
Some lines below should be removed, on top of it (could not add to the suggestion....